app/src/util_ios.h

```diff
@@ -188,7 +188,7 @@ typedef BOOL (
 // Call the given block once for every Objective-C class that exists that
```

Comment:
This comment should be changed to reflect the new behavior - calling the block for every objective-C class that's passed into [UIApplication setDelegate:]